### DISCUSSION TITLE
Sourcing gw_setup.sh means you can no longer use "tab" to fill out file names or commands

### DISCUSSION THREAD

**Discussion by JessicaMeixner-NOAA** [NOAA-EMC/global-workflow#1937] [Q&A]
After I source gw_setup.sh and it loads the set up modules I no longer can use "tab" to fill out commands or file names.   Is there anyway to have that capability re-enabled other than starting a new terminal?

**Comment by aerorahul** [comment]
Do `set +u`after you source this script to get around this issue.
We will fix the root cause at some point.  It has been noted. 
@WalterKolczynski-NOAA Can we issue a quick hotfix?

**Reply by JessicaMeixner-NOAA** [reply]
Thank you @aerorahul !!!

**Comment by aerorahul** [comment]
#2046 fixed this issue.
